components/server/ee/src/workspace/gitpod-server-impl.ts

```diff
@@ -2370,17 +2376,33 @@ export class GitpodServerEEImpl extends GitpodServerImpl {
     }
 
     async listUsage(ctx: TraceContext, req: ListUsageRequest): Promise<ListUsageResponse> {
-        const { from, to } = req;
         const attributionId = AttributionId.parse(req.attributionId);
         if (!attributionId) {
             throw new ResponseError(ErrorCodes.INVALID_COST_CENTER, "Bad attribution ID", {
                 attributionId: req.attributionId,
             });
         }
-        traceAPIParams(ctx, { attributionId });
         const user = this.checkAndBlockUser("listUsage");
         await this.guardCostCenterAccess(ctx, user.id, attributionId, "get");
+        return this.internalListUsage(ctx, req);
+    }
+
+    async getUsageBalance(ctx: TraceContext, attributionId: string): Promise<number> {
+        const user = this.checkAndBlockUser("listUsage");
+        await this.guardCostCenterAccess(ctx, user.id, AttributionId.parse(attributionId) as AttributionId, "get");
```

Comment:
nit: missing check for parsed `attributionId` as it's done in in other RPC methods.